BraintreeDropIn/BTDropInBaseViewController.m

```diff
@@ -19,7 +19,7 @@ @implementation BTDropInBaseViewController
 - (instancetype)initWithAPIClient:(BTAPIClient *)apiClient request:(BTDropInRequest *)request
 {
     if (self = [super init]) {
-        self.apiClient = [apiClient copyWithSource:apiClient.metadata.source integration:BTClientMetadataIntegrationDropIn2];
+        self.apiClient = apiClient;
```

Comment:
This lets us access the cached configuration fetched previously, instead of having to re-fetch it for each view controller.

Reply:
I think this will change the [metadata](https://github.com/braintree/braintree_ios/blob/2946eebfdef0729dade1f7994ff1869145a04a1d/Sources/BraintreeCore/BTClientMetadata.m#L61-L72) sent in Drop-In's analytics requests. We send `self.apiClient.metadata.integrationString` throughout the Base SDK.

Reply:
The metadata is being set on the api client when it's initially passed into the `BTDropInController`, [here](https://github.com/braintree/braintree-ios-drop-in/blob/4ece967437cf6db3db299d10c0f3120d44391d93/BraintreeDropIn/BTDropInController.m#L61). Once it's been set once, we shouldn't need to continue setting it.